openvpn/tunnel/setup_tun_linux.go

```diff
@@ -176,6 +154,14 @@ func findFreeTunDevice() (tun tunDevice, err error) {
 	return tun, ErrNoFreeTunDevice
 }
 
+func (service *LinuxTunDeviceManager) deviceExists(tunName string) (exists bool, err error) {
```

Comment:
It returns `error` but we are not returning any error inside the function.

Reply:
good catch, fixed.